terraform/shared/ecr.tf

```diff
@@ -33,6 +33,14 @@ resource "aws_ecr_repository" "api" {
   }
 }
 
+resource "aws_ecr_repository" "concepts" {
+  name = "uk.ac.wellcome/concepts"
```

Comment:
weco/concepts?

Or does that wreak havoc with weco-deploy?

Reply:
Also, is this an opportunity to start defining some better defaults around ECR repository policies for expiring old images?

Reply:
Yeah, this naming is to satisfy weco-deploy unfortunately.

I was thinking about repo policies too - yes, we should do that, but not as part of this PR. Standardising the policies won't be harder if we have an extra repo, and I don't want this effort to get a service up and running quickly to accrue wider improvements as we do it. Rather, we should be tracking the things that it makes us notice and then fixing them as we see fit: https://github.com/wellcomecollection/platform/issues/5546

Reply:
Sounds good on both counts. 👍